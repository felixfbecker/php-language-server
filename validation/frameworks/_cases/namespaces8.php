<?php
declare(strict_types = 1);

namespace LanguageServer\Tests\Utils;

use function LanguageServer\{pathToUri, uriToPath};