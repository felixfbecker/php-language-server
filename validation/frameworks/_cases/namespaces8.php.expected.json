{
    "references": {
        "LanguageServer": [
            "./_cases/namespaces8.php"
        ]
    },
    "definitions": {
        "LanguageServer\\Tests\\Utils": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "LanguageServer\\Tests\\Utils",
                "kind": 3,
                "location": {
                    "uri": "./_cases/namespaces8.php"
                },
                "containerName": "LanguageServer\\Tests"
            },
            "type": null,
            "documentation": null
        }
    }
}